tests/unit/test_orders.py

```diff
@@ -1,30 +1,61 @@
-"""Tests of Orders client."""
+"""Tests of Orders client and aiohttp implementation."""
 
-from unittest.mock import patch
+from unittest.mock import Mock, patch
 
 import pytest
 
-import planet.clients.orders
-from planet.clients.orders import OrdersClient
-from planet.order_request import build_request, product
+from planet.clients.orders import OrdersClient, OrderError
 
 
 @pytest.mark.asyncio
 @patch("planet.clients.orders.ClientSession.post")
-async def test_create_order(mock_post, event_loop):
-    """Create an order."""
-    image_ids = ['3949357_1454705_2020-12-01_241c']
-    request = build_request('test_order',
-                            [product(image_ids, 'analytic', 'psorthotile')])
-
-    mock_post.return_value.__aenter__.return_value.json.return_value = {
-        "id": "lol",
-        "state": "wut"
+async def test_create_order_failure(mock_http_post):
+    """Get expected chained exception on fake service failure."""
+
+    # create_order() interacts with the Planet Orders service only via
+    # HTTP POST request. Here we patch the post() method of the aiohttp
+    # ClientSession class and program the mock object to raise an
+    # exception when its raise_for_status() method is called. Note that
+    # raise_for_status() is a normal (not-async) Python method and so
+    # we must explicitly use (not-async) Mock. Otherwise all the
+    # attributes of mock_http_post will be AsyncMock objects.
```

Comment:
In test_http.py we use respx, which is neat. In this PR, the approach to faking a POST response is more direct and low level.